### ISSUE TITLE
The Outer Worlds: Spacer's Choice Edition does not start

### ISSUE THREAD

**Issue by pamidur** [doitsujin/dxvk#3778]
Game does not start.

### Software information
The Outer Worlds: Spacer's Choice Edition
This is dx11 game, but uses 11on12, so vkd3d is required

### System information
- GPU: 6900xt
- Driver: 2.0.283
- Windows version:  11
- DXVK version: 2.3
- vkd3d version: 2.11.1

### Apitrace file(s)
Game does not start with APItrace. Two traces in the zip - with and without vkd3d
[IndianaEpicGameStore-Win64-Shipping.zip](https://github.com/doitsujin/dxvk/files/13803958/IndianaEpicGameStore-Win64-Shipping.zip)

### Log files
Log are slightly different when I set `dxgi.hideAmdGpu = True`
[hide amd gpu.zip](https://github.com/doitsujin/dxvk/files/13803963/hide.amd.gpu.zip)
[just dlls.zip](https://github.com/doitsujin/dxvk/files/13803964/just.dlls.zip)

### TLDR
dxvk + vkd3d
```
[2024.01.01-07.34.56:209][  0]LogWindows: Error: Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x0000000000000038
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ffc4009d3e9 amdxc64.dll!UnknownFunction []
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ffc4009e264 amdxc64.dll!UnknownFunction []
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff7563ae46b IndianaEpicGameStore-Win64-Shipping.exe!InitDX12Extensions() []
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff7563aee9f IndianaEpicGameStore-Win64-Shipping.exe!agsDriverExtensionsDX12_CreateDevice() []
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff754339d6c IndianaEpicGameStore-Win64-Shipping.exe!FD3D12Adapter::CreateRootDevice() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\D3D12RHI\Private\D3D12Adapter.cpp:414]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff754341921 IndianaEpicGameStore-Win64-Shipping.exe!FD3D12Adapter::InitializeDevices() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\D3D12RHI\Private\D3D12Adapter.cpp:770]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff7543a27f0 IndianaEpicGameStore-Win64-Shipping.exe!FD3D12DynamicRHI::Init() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\D3D12RHI\Private\Windows\WindowsD3D12Device.cpp:666]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff7550132ee IndianaEpicGameStore-Win64-Shipping.exe!RHIInit() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\RHI\Private\DynamicRHI.cpp:229]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff752cc274c IndianaEpicGameStore-Win64-Shipping.exe!FEngineLoop::PreInitPreStartupScreen() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\Launch\Private\LaunchEngineLoop.cpp:2631]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff752cbacc6 IndianaEpicGameStore-Win64-Shipping.exe!GuardedMain() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\Launch\Private\Launch.cpp:132]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff752cbaffa IndianaEpicGameStore-Win64-Shipping.exe!GuardedMainWrapper() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:144]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff752cbc3f4 IndianaEpicGameStore-Win64-Shipping.exe!LaunchWindowsStartup() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:301]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff752cc5b14 IndianaEpicGameStore-Win64-Shipping.exe!WinMain() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\Launch\Private\Windows\LaunchWindows.cpp:349]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff7568056ee IndianaEpicGameStore-Win64-Shipping.exe!__scrt_common_main_seh() [D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ffd076e257d KERNEL32.DLL!UnknownFunction []
[2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ffd07ccaa58 ntdll.dll!UnknownFunction []
[2024.01.01-07.34.56:209][  0]LogWindows: Error: 
[2024.01.01-07.34.56:221][  0]LogExit: Executing StaticShutdownAfterError
[2024.01.01-07.34.56:221][  0]LogWindows: FPlatformMisc::RequestExit(1)
[2024.01.01-07.34.56:221][  0]LogWindows: FPlatformMisc::RequestExitWithStatus(1, 3)
[2024.01.01-07.34.56:221][  0]LogCore: Engine exit requested (reason: Win RequestExit)
```
dxvk + vkd3d +  `dxgi.hideAmdGpu = True`
```
[2024.01.01-07.37.02:068][  0]LogThreadingWindows: Error: Runnable thread TaskGraphThreadHP 16 crashed.
[2024.01.01-07.37.02:068][  0]LogThreadingWindows: Error: Runnable thread TaskGraphThreadHP 14 crashed.
[2024.01.01-07.37.02:068][  0]LogWindows: Error: === Critical error: ===
[2024.01.01-07.37.02:068][  0]LogWindows: Error: 
[2024.01.01-07.37.02:068][  0]LogWindows: Error: HandleError re-entered.
[2024.01.01-07.37.02:068][  0]LogWindows: FPlatformMisc::RequestExit(1)
[2024.01.01-07.37.02:068][  0]LogWindows: Error: Fatal error!
[2024.01.01-07.37.02:068][  0]LogWindows: FPlatformMisc::RequestExitWithStatus(1, 3)
[2024.01.01-07.37.02:068][  0]LogWindows: Error: Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x00000000000001c0
[2024.01.01-07.37.02:068][  0]LogWindows: Error: 
[2024.01.01-07.37.02:068][  0]LogWindows: Error: 
[2024.01.01-07.37.02:068][  0]LogWindows: Error: [Callstack] 0x00007ffc45234a70 amdvlk64.dll!UnknownFunction []
[2024.01.01-07.37.02:068][  0]LogCore: Engine exit requested (reason: Win RequestExit)
[2024.01.01-07.37.02:068][  0]LogWindows: Error: [Callstack] 0x00007ffc46c4d3e7 amdvlk64.dll!UnknownFunction []
[2024.01.01-07.37.02:068][  0]LogWindows: Error: [Callstack] 0x00007ffc46c44c38 amdvlk64.dll!UnknownFunction []
```

**Comment by pamidur** [comment]
I believe #3773 was mistakenly closed.

**Comment by mbriar** [comment]
It's an unreal engine 4 game, why do you think it uses d3d11on12? It just uses the ue4 d3d13 rhi. Anyways, the game works fine on linux with dxvk+vkd3d-proton, so i doubt this is a problem within dxvk's control.

**Comment by mbriar** [comment]
Also you should post the vkd3d-proton log as well, because dxvk isn't even used for rendering in this game. But it most likely just an AMD windows vulkan driver bug, it's regularly broken and unusable with vkd3d-proton.

**Comment by K0bin** [comment]
> [2024.01.01-07.34.56:209][  0]LogWindows: Error: [Callstack] 0x00007ff754339d6c IndianaEpicGameStore-Win64-Shipping.exe!FD3D12Adapter::CreateRootDevice() [F:\Projects\VTS.GEN9.STAGE\Game\Engine\Source\Runtime\D3D12RHI\Private\D3D12Adapter.cpp:414]

This is using D3D12.

**Comment by kleinermob** [comment]
same issue, IndianaEpicGameStore-Win64-Shipping.exe just stays as "inactive" in task manager with 600mb ram used, its shown in the tasbar but no window actually appears.

**Comment by kleinermob** [comment]
with -dx11 launch options its actually directx 11
